src/style-spec/reference/v8.json

```diff
@@ -652,7 +652,7 @@
     },
     "filter": {
       "type": "filter",
-      "doc": "A expression specifying conditions on source features. Only features that match the filter are displayed. Zoom expressions in filters are only evaluated at integer zoom levels. The `feature-state` expression is not supported in filter expressions."
+      "doc": "An expression specifying conditions on source features. Only features that match the filter are displayed. Zoom expressions in filters are only evaluated at integer zoom levels. The `feature-state` expression is not supported in filter expressions.  The `['pitch']` and `['distance-from-center']` expressions are supported only for filter expressions on the `symbol` layer."
```

Comment:
```suggestion
      "doc": "An expression specifying conditions on source features. Only features that match the filter are displayed. [`zoom`](#zoom) expressions in filters are only evaluated at integer zoom levels. The [`feature-state`](#feature-state) expression is not supported in filter expressions.  The [`pitch'](#pitch) and [`distance-from-center'`](distance-from-center) expressions are supported only for filter expressions on the [`symbol`](/mapbox-gl-js/style-spec/layers/#symbol) layer."
```

Reply:
This is a good idea but i don't think we should do this manually, instead the docs codegen system should handle this, that way if the structure of the docs website changes it can account for it.

Reply:
That sounds like a good idea to me!